File: Java/Numbers/program7.java
// Program to display 5 times Hello on screen

import java.util.*;

class Number
{
    public void Display(int iNo)
    {
        int iCnt = 0;

        for(iCnt = 1; iCnt <= iNo; ++iCnt)
        {
            System.out.println("Hello");
        }
    }
}

class program7
{
    public static void main(String b[])
    {
        int iValue = 0;

        Scanner sobj = new Scanner(System.in);
        
        System.out.println("Enter the number");
        iValue = sobj.nextInt();

        Number obj = new Number();

        obj.Display(iValue);
    }
}

File: C++/Number/program7.cpp
// Program to display N times Hello on screen

#include<iostream>
using namespace std;

class Number
{
    public:
        void Display(int iNo)
        {
            int iCnt = 0;

            for(iCnt = 1; iCnt <= iNo; ++iCnt)
            {
                cout<<"Hello"<<endl;
            }
            
        }
};
    

int main()
{
    Number nobj;

    int iValue = 0;

    cout<<"Enter the number"<<endl;
    cin>>iValue;

    nobj.Display(iValue);

    return 0;
}
